backend/routes/admin.py

```
@@ -193,10 +206,14 @@ async def change_password(
         )
 
-        # Expire all sessions for this user (except current one)
+        # SECURITY FIX: Force complete re-authentication after password change
+        # This prevents session fixation attacks
         admin_auth_manager.expire_user_sessions(user["id"])
 
-        return {"success": True, "message": "Password changed successfully"}
+        # Force logout by clearing the current session cookie
+        response = JSONResponse({"success": True, "message": "Password changed successfully. Please log in again."})
+        response.delete_cookie("admin_session", path="/", httponly=True, secure=True, samesite="lax")
+        return response
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)